src/parquet/schema.cc

```diff
@@ -113,7 +113,7 @@ PrimitiveNode::PrimitiveNode(const std::string& name, Repetition::type repetitio
 
   // PARQUET-842: In an earlier revision, decimal_metadata_.isset was being
   // set to true, but Impala will raise an incompatible metadata in such cases
-  memset(&decimal_metadata_, 0, sizeof(decimal_metadata_));
+  std::memset(&decimal_metadata_, 0, sizeof(decimal_metadata_));
```

Comment:
Don't use `std::` here. People reported problems on the usage of `std::memset` (seems like some BSD variants don't have this).

Reply:
Thanks for the heads up, will change.